CLAUDE.md

```
@@ -0,0 +1,1667 @@
+|------|---------|
+| `tests/test_autowire.py` | Core auto-wire tests (260 lines) |
+| `tests/test_autowire_unittest.py` | Additional unit tests |
+
+---
+
+## Dependencies
+
+### Python Dependencies
+
+**Core Runtime**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)